src/pages/computer-science/what-is-an-sdk/index.md

```diff
@@ -8,7 +8,7 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
-The term "SDK" stands for Sofware Development Kit.
+
```

Comment:
This removes the content added to this Guide. In this case, I'm okay with that.

Reply:
👍 That's fair, it's not adding much